core-common/src/main/java/org/mapstruct/MapperConfig.java

```diff
@@ -135,4 +137,13 @@
      */
     MappingInheritanceStrategy mappingInheritanceStrategy()
         default MappingInheritanceStrategy.EXPLICIT;
+
+    /**
+     * Determines when to include a null check on the source property value of a bean mapping.
+     *
+     * Can be overridden by the one on {@link Mapper} or {@link Mapping}.
+     *
+     * @return strategy how to do null checking
+     */
+    SourceValuePresenceCheckStrategy sourceValuePresenceCheckStrategy() default ON_IMPLICIT_CONVERSION;
```

Comment:
Somewhere here we should mention that a value presence check method will take precedence if present, and that this setting will not apply in this case.